### ISSUE TITLE
Cant boot pop os on a asus tp202nas

### ISSUE THREAD

**Issue by jagdtigger** [pop-os/pop#512]
**Distribution (run `cat /etc/os-release`):**
 
 
 
**Related Application and/or Package Version (run `apt policy $PACKAGE NAME`):**

**Issue/Bug Description:**
No matter which thumb drive or program i use to make the drive bootable it will always freeze with a underscore on the screen(it blinks once). Secure boot disabled.

**Steps to reproduce (if you know):**

**Expected behavior:**

**Other Notes:**
Machine: Asus TP202NAS
pop is version: tried 18.04, 18.10, 19.04

**Comment by kafran** [comment]
Try to use https://www.balena.io/etcher/ to make your flashdrive.

**Comment by jagdtigger** [comment]
I tied that, along with unetbootin and rufus...

**Comment by kafran** [comment]
I was facing this problem on the last Pop_OS version. But 19.04 is working for me now.

**Comment by jagdtigger** [comment]
It seems there is a new iso, will try it then report back...  Is there a way to get any sort of logfile/info to pinpoint where its freezing?

**Comment by asadana** [comment]
I'm actually seeing the exact same behavior with Pop_OS! 19.04 on my Dell 5530 i5 with integrated Intel UHD 630, no discrete GPU.
It's worth noting that Ubuntu 19.04 boots and works perfectly on the same hardware for me.

**Comment by jagdtigger** [comment]
Yep, (almost the) same here.... (It has some issues with the eMMC when it switches to HS400 speed.)

**Comment by kevhemingway88** [comment]
still no solution
see this:
https://www.reddit.com/r/pop_os/comments/dkgqd4/6_months_later_still_cant_boot_with_any_version/
